File: app/src/main/java/openfoodfacts/github/scrachx/openfood/models/HeaderNutrimentListItem.java
package openfoodfacts.github.scrachx.openfood.models;

import androidx.annotation.NonNull;

import static openfoodfacts.github.scrachx.openfood.utils.Utils.bold;

/**
 * Group of NutrimentListItem with bold values
 */
public class HeaderNutrimentListItem extends NutrimentListItem {
    /**
     * Header with bold values
     * @param title
     * @param value
     * @param servingValue
     * @param unit
     */
    public HeaderNutrimentListItem(@NonNull CharSequence title, @NonNull CharSequence value,
                                   @NonNull CharSequence servingValue, @NonNull CharSequence unit,
                                   @NonNull CharSequence modifier) {
        super(bold(title), bold(value), bold(servingValue), bold(unit), bold(modifier));
    }


    /**
     * Header with only bold title
     * @param title
     */
    public HeaderNutrimentListItem(@NonNull CharSequence title) {
        super(bold(title), "", "", "", "");
    }
}

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/models/HeaderNutrimentListItem.kt
package openfoodfacts.github.scrachx.openfood.models

import openfoodfacts.github.scrachx.openfood.utils.bold

/**
 * Group of NutrimentListItem with bold values
 */
class HeaderNutrimentListItem : NutrimentListItem {
    /**
     * Header with bold values
     * @param title
     * @param value
     * @param servingValue
     * @param unit
     */
    constructor(
            title: CharSequence,
            value: CharSequence,
            servingValue: CharSequence,
            unit: CharSequence,
            modifier: CharSequence
    ) : super(bold(title), bold(value), bold(servingValue), bold(unit), bold(modifier))

    /**
     * Header with only bold title
     * @param title
     */
    constructor(title: CharSequence) : super(bold(title), "", "", "", "")
}